### ISSUE TITLE
Add support for searching authors in ORCiD at submission

### ISSUE THREAD

**Issue by mauromsl** [BirkbeckCTP/janeway#973]
**Is your feature request related to a problem? Please describe.**
When adding an author to an article, Janeway supports searching by ORCiD, where it will look up the janeway DB. We could also search ORCiD database and import the author into Janeway (pre-populating as many author fields as possible from ORCiD data)

**Comment by ajrbyers** [comment]
This will be covered in https://github.com/BirkbeckCTP/janeway/issues/1010

**Comment by amardesi** [comment]
This approach is discouraged and can lead to the wrong ORCID Ids being assigned. Instead, I'd like to have user enter email of author which will request that author connects to Janeway with their ORCID ID in order to accept assignment of authorship. See bullet #1 here: https://orcid.org/organizations/publishers/best-practices and this: https://members.orcid.org/api/resources/find-myresearchers

**Comment by ajrbyers** [comment]
I don’t think this issue was suggesting anyone search by name for an author in the orcid database, simply that if you know someone’s orcid that it can be used to prepopulate information in the author record rather than retyping it...

**Comment by alainna** [comment]
hi @ajrbyers! Alainna here (I'm now with CDL and previously I was with ORCID, working closely with publishing platforms). It is strongly recommended that ORCID iDs never be allowed to be input manually -- not by individual users, not by journal editors, and not by administrators. 

Just because someone writes to you saying that ORCID iD https://orcid.org/0000-0001-2345-6789 is their ORCID iD does not necessarily mean that it is indeed their iD. It could be one by someone with the same name, or it could be an ORCID iD created on their behalf which they do not control, or the person submitting the article could create a new ORCID iD on the author's behalf, not knowing that the author already has an ORCID iD. Until a researcher authenticates their ORCID iD, there's no way to know that the researcher is indeed in control of it. There's a nice blog post from a few years back on this: [What's So Special About Signing In?](https://orcid.org/blog/2017/02/20/whats-so-special-about-signing) 

To share a personal anecdote: Someone once manually input [my personal ORCID iD](https://orcid.org/0000-0002-6138-0903) when submitting a journal article (possibly they received it from my email signature during an ORCID support query). As a result, my ORCID iD is associated with her work: http://digital.csic.es/bitstream/10261/161507/1/GMHresena.pdf There wasn't a DOI created for the article, so I didn't find out that someone had mistakenly associated my ORCID iD with their work in the way that most researchers do. In those cases, researchers receive a notification from Crossref requesting permission to add an article that the researcher did not write to their ORCID record, all because the wrong ORCID iD was added to the article and thus DOI. The researcher then has to jump through hoops to get their ORCID iD removed from the published article and its DOI metadata -- as well as other systems, such as the manuscript submission system -- in order to be able to use Crossref auto-update or other services as expected. It's not a fun thing to go through. 🙁 

TL;DR: Authentication is awesome because it's the only way to know the user is in control of that ORCID iD. Yay authentication! 😸

**Comment by ajrbyers** [comment]
Hi @alainna, I don't have a problem with the workflow that has been suggested, I was merely closing this issue because the workflow suggested by @amardesi covers what was proposed here in the recommended fashion. I would still like for data to be pre-populated by ORCID so even if you invite someone by email and they connect via ORCID we can pull data from the API to complete fields?

For clarity, we should also make this optional, so if someone is invited to accept authorship of the paper we shouldn't force them to connect their ORCID.

**Comment by ajrbyers** [comment]
In fact, assuming we don't force people to connect their ORCID this will also cover https://github.com/BirkbeckCTP/janeway/issues/145

**Comment by amardesi** [comment]
Great, thanks for the clarification @ajrbyers
